sql/conseil.sql

```diff
@@ -284,6 +284,7 @@ CREATE TABLE tezos.operations (
     amount numeric,
     destination character varying,
     parameters character varying,
+    parameters_entrypoint character varying,
```

Comment:
Since a transaction can apparently have multiple parameters, we should rename this field to `parameters_entrypoints`.

Reply:
I think that to add multiple entrypoints with related scripts, we also need to modify the Model that represents a transaction, for now it doesn't support an array of parameters anyway...

Reply:
@vishakh what do you think about leaving it as `_entrypoint` instead of `_entrypoints` as @ivanopagano suggested?

Reply:
I think `_entrypoints` is more future proof.

Reply:
ok, changed